### PR TITLE
doc: add use example in readme

### PR DESCRIPTION
**Pull request by mrrtree**
# Which issue does this PR close?

Closes #

# Rationale for this change
 
 

- Add example in readme

# What changes are included in this PR?

# Are there any user-facing changes?

# How does this change test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,114 @@
...
+## Support features
+- Manage
+  - creating and alter table
+- Query data
+  - using sql
+- Write data
+  - Point build
 
-# Contributing
+## How To Use
```
Comment (jiacai2050) [inline README.md:15]: I prefer to add this usage to docs, and link there, otherwise we will have to maintain two copies

Comment (mrrtree) [reply README.md:15]: The content if copy from docs, i think the readme also needs a simple example.
but according to your suggestion, we can add a link here to jump to doc, keep readme simple

Comment (mrrtree) [reply README.md:15]: add link with docs, how about doc github link or https://docs.ceresdb.io/?

File: README.md [hunk 2]
```diff
@@ -4,16 +4,24 @@
...
+
+Reader our [golang SDK docs](https://docs.ceresdb.io/en/sdk/go.html)
```
Comment (jiacai2050) [inline README.md]:
```suggestion
Please refer [Golang SDK docs](https://docs.ceresdb.io/en/sdk/go.html).
```

**Review by jiacai2050 — APPROVED** [review]
LGTM

**Pull request merged**